### PR TITLE
Removing Copy Button on Search Results page

### PR DESCRIPTION
**Pull request by naushervan**
1. Resolution of issue #88 by removing copy button from Search Results page.
2. Fixing development environment error
3. When we pressed copy button caret to close copy menu, and then pressed it again to open it, then the menu opened and then closed automatically. This issue was fixed.

### CODE REVIEW

File: application/components/search/SearchEngine.php [hunk 1]
```diff
@@ -36,7 +36,7 @@ public function doSearch($query)
         $resultset = $this->doSearchInternal();
         if ($resultset === null) {
             $this->doNotifyOutage();
-        } else {
+        } else if (YII_ENV != 'dev') {
```
Comment (ahadith) [inline application/components/search/SearchEngine.php:39]: Can you please add a `defined("YII_ENV")` check here before comparing its value? Make sure that if the stage variable is prod it still works.

File: application/modules/front/views/collection/hadith_reference.php [hunk 2]
```diff
@@ -149,9 +149,8 @@
...
 				//echo "<a href=\"$permalink\">Permalink</a>";
 			if (!isset($hideReportError) or !$hideReportError) echo "<span class=reportlink href=\"javascript: void(0);\" onclick=\"reportHadith(".$urn.", '".$divName."')\">Report Error</span> | ";
-			if (!$hideShare) echo  "<span class=sharelink onclick=\"share('$permalink')\">Share</span> | "; 
-				echo "<span class=copylink title=\"Copy hadith to clipboard\">Copy</span> "; 
-				echo "<span class=copycbcaret title=\"Change copy options\">▼</span>".
-			"</div>". 
-		"</div>";
+			if (!$hideShare) echo "<span class=sharelink onclick=\"share('$permalink')\">Share</span> | "; 
+			if (!$hideShare) echo "<span class=copylink title=\"Copy hadith to clipboard\">Copy</span> "; 
+			if (!$hideShare) echo "<span class=copycbcaret title=\"Change copy options\">▼</span>";
```
Comment (ahadith) [inline application/modules/front/views/collection/hadith_reference.php:154]: Can these three echo statements be put in the same `if` block?

File: application/components/search/SearchEngine.php (hunk 1 above)
Comment (naushervan) [reply application/components/search/SearchEngine.php:39]: I have done it like this:
`        
else if (defined("YII_ENV") and YII_ENV != 'dev') {
    $this->logQuery($resultset->getCount());
}
`
If I convert the stage variable in .env.local config file to `prod`, it gives error:
`An internal server error occurred.`

File: application/modules/front/views/collection/hadith_reference.php (hunk 2 above)
Comment (naushervan) [reply application/modules/front/views/collection/hadith_reference.php:154]: Done.

**Pull request merged**